lawlibrary/views/home.py

```diff
@@ -13,5 +13,6 @@ def get(self, request, *args, **kwargs):
         context["recent_legislation"] = Legislation.objects.filter(
             metadata_json__stub=False
         ).order_by("-date")[:10]
+        context["recent_articles"] = Article.objects.order_by("-date")[:5]
```

Comment:
For now, only africanlii has any need of articles, so this and the changes to home.html and header.html can go into africanlii.